crates/turbopack-core/src/chunk/mod.rs

```diff
@@ -27,9 +27,8 @@ use crate::{
 };
 
 /// A module id, which can be a number or string
-#[turbo_tasks::value(shared)]
+#[turbo_tasks::value(shared, serialization = "custom")]
 #[derive(Debug, Clone, Hash, DeterministicHash)]
-#[serde(untagged)]
```

Comment:
Shouldn't that be the exact same behavior?

Reply:
Yeah. I think the same.